server/sandbox_network.go

```diff
@@ -132,10 +133,10 @@ func (s *Server) networkStop(ctx context.Context, sb *sandbox.Sandbox) {
 	podNetwork, err := s.newPodNetwork(sb)
 	if err != nil {
 		log.Warnf(ctx, err.Error())
-		return
+		return nil
```

Comment:
We need to return this error as the next call will fail if this doesn't succeed.